django_celery_beat/models.py

```diff
@@ -28,13 +28,61 @@
     (MICROSECONDS, _('Microseconds')),
 )
 
+SOLAR_SCHEDULES = [(x, _(x)) for x in schedules.solar._all_events]
+
 
 def cronexp(field):
     """Representation of cron expression."""
     return field and str(field).replace(' ', '') or '*'
 
 
 @python_2_unicode_compatible
+class SolarSchedule(models.Model):
+    """Schedule following astronomical patterns."""
+
+    event = models.CharField(
+        _('event'), max_length=24, choices=SOLAR_SCHEDULES
+    )
+    latitude = models.DecimalField(
+        _('latitude'), max_digits=9, decimal_places=6
+    )
+    longitude = models.DecimalField(
+        _('latitude'), max_digits=9, decimal_places=6
+    )
+
+    class Meta:
+        """Table information."""
+
+        verbose_name = _('solar')
+        verbose_name_plural = _('solars')
```

Comment:
sorry @ask, i forgot to check travis last night. the `unique_together` has been added.